CHANGELOG.rst

```diff
@@ -4,6 +4,23 @@ Changelog
 
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
+1.2.1 (2021-11-22)
+------------------
+
+**Added**
+
+**Fixed**
+
+* Introduces proper HTTP response formatting for unauthorized requests (401) by providing the
+WWW-Authentication header entry. This will enable clients that implement the HTTP specification strictly
+to be able to make a successful request to this REST server
+
+**Dependencies**
+
+* Micronaut 1.2.5 -> 2.5.13
+* Groovy 2.5.x -> 3.0.5
```

Comment:
```suggestion
* Groovy 2.5.10 -> 3.0.5
```

Reply:
You could also add here that new dependencies have been added like micronaut-jdbc-dbcp and javax.annotation-api and that micronaut-utils-lib was removed. The changes in the pom seem important so they should be reflected here.

Reply:
I think it was not only 2.5.10. As this is already in the release I would leave it as is. Thanks for pointing it out though

Reply:
We should consider looking out for that more strictly. Crucial changes like should be reflected IMO.